src/main/java/com/google/sps/servlets/InfoCardServlet.java

```diff
@@ -59,20 +72,77 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity chosenElection = ServletHelper.findElectionInDatastore(datastore, electionId);
+
+    if (chosenElection == null) {
```

Comment:
Add this to the ServletHelper class and generalize to other Datastore entity types?